website/content/en/docs/AWS/launch-templates.md

```diff
@@ -41,6 +41,22 @@ AMI (Amazon Machine Image), is the base image/VM for a launch template.
 [Review the instructions for importing a VM to AWS.](https://docs.aws.amazon.com/vm-import/latest/userguide/vmimport-image-import.html) Note the AMI id generated by this process, such as,
 `ami-074cce78125f09d61`.
 
+### Instance Profile - IAM
+
+The launch template must include an "instance profile" -- an IAM role.
+
+The instance profile must include *at least* the permissions of the default Karpenter node instance profile. See the default role, `KarpenterNodeRole`, in the full example below for more information.
+
+See also, [the managed policy "AmazonEKSWorkerNodePolicy"](https://docs.aws.amazon.com/eks/latest/userguide/security-iam-awsmanpol.html#security-iam-awsmanpol-AmazonEKSWorkerNodePolicy) which includes permission to describe clusters and subnets.
+
+### Tags 
+
+ ✳️ Note: this requirement only applies to EKS clusters. 
+
+Instances must have the tag `kubernetes.io/cluster/EksClusterName` and value `owned` to successfully join an EKS cluster. 

```

Comment:
Mentioned earlier. This has nothing to do with EKS.

Reply:
This tag isn't required with self managed clusters.

Reply:
Did you confirm this one? If so, happy to merge.